testdata/testoutput5

```diff
@@ -6238,9 +6238,10 @@ Failed: error 115 at offset 52: reference to non-existent subpattern
 ------------------------------------------------------------------
         Bra
         eclass[
-          cls:[\p{Lu}]
-          cls:[\p{Nd}]
-          op: ||
+          bitmap: [0-9A-Z\xc0-\xd6\xd8-\xde]
```

Comment:
I think this is exactly the case where no bitmap is needed.

Reply:
Yes, sadly. Because ECLASS combines bitmaps in ways that are harder to reason about, I made the ECLASS bitmap behaviour a bit different to XCLASS (see the changes in HACKING). For ECLASS, I made it so that there is a bitmap if and only if the ECLASS matches any characters < 256. So an ECLASS will have at most one bitmap, but it will be preserved for cases like `\pL & \pN`.

(This example is even worse, because additional optimisation would actually mean it could be simplified to a simple XCLASS.)

Reply:
Ok. My idea was to detect that bitmap is needed in the `lengthptr != NULL` phase, and use this info later. I will try to do it after this patch is landed.